### PR TITLE
update date from 2020 to 2021 on README file

### PR DESCRIPTION
**Pull request by innoCyber**
### First time contributor checklist

- [x]  I have read [how to contribute](https://github.com/signalapp/Signal-Android/blob/master/CONTRIBUTING.md) to this project
- [x]  I have signed the [Contributor License Agreement](https://whispersystems.org/cla/)

### Contributor checklist

- [x]  I am following the [Code Style Guidelines](https://github.com/signalapp/Signal-Android/wiki/Code-Style-Guidelines)
- [x]  I have tested my contribution on these devices:
 * Samsung Galaxy Note 9, Android 9
 * Samsung A20, Android 9
 * Virtual device Google Pixel 4, Android 10
- [x] My contribution is fully baked and ready to be merged as is
- [x] I ensure that all the open issues my contribution fixes are mentioned in the commit message of my first commit using the `Fixes #1234` [syntax](https://help.github.com/articles/closing-issues-via-commit-messages/)

----------

### CODE REVIEW

**Review by RHeynsZa — APPROVED** [review]

**Review by johan-marx — APPROVED** [review]
SeemsGood

**Comment by EwoutH** [comment]
I appreciate that you tested a non-function change to a documentation file on 3 different devices 💯

**Comment by innoCyber** [comment]
> I appreciate that you tested a non-function change to a documentation file on 3 different devices 💯

This feels like a banter 🤣🤣

**Review by Relluem94 — APPROVED** [review]

**Review by bitactro — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ The form and manner of this distribution makes it eligible for export under the
 
 ## License
 
-Copyright 2013-2020 Signal
+Copyright 2013-2021 Signal
```
Comment (x80486) [inline README.md:62]: Wouldn't be better to "say" `2013 - Present` so this change doesn't have to be done on every calendar year?

**Pull request closed without merging**